keylime/da/record.py

```diff
@@ -356,7 +356,7 @@ def base_build_key_list(registration_record_list: list) -> list:
     return aik_list
 
 
-def get_record_mgt_class(store_import: bool = False):
+def get_record_mgt_class(store_import: str = ""):
```

Comment:
This is fine because strings are not mutable in Python, but in a general case this can be a red flag